package.json

```diff
@@ -47,10 +48,10 @@
     "webpack-cli": "^3.1.2",
     "webpack-dev-server": "^3.1.14",
     "xmlhttprequest": "^1.8.0",
-    "yargs": "^12.0.5",
-    "dotenv": "^5.0.1"
+    "yargs": "^12.0.5"
   },
   "dependencies": {
+    "@babel/polyfill": "^7.2.5",
```

Comment:
I think this polyfill is deprecated? it certainly isn't the recommended way to polyfill. 
Babel now mentions `core-js` in it's doc's, and the way I've done it in a number of projects is to include the whole of core-js.  True, we can polyfill only what is needed, but to get going quickly, you could also do this.

```js
// https://babeljs.io/blog/2019/03/19/7.4.0
// the following will add _all_ polyfills. Eat that ie11! (35kb)
import 'core-js/stable';
import 'regenerator-runtime/runtime';
import 'whatwg-fetch';
```

Reply:
Has this PR any downsides e.g. problems on other browsers?

Reply:
not that i know off, so certainly not a blocker